apps/service-portal/src/utils/userManager.ts

```diff
@@ -4,9 +4,10 @@ import {
   InMemoryWebStorage,
   UserManagerSettings,
 } from 'oidc-client'
+import { environment } from '../environments'
 
 const settings: UserManagerSettings = {
-  authority: 'https://siidentityserverweb20200805020732.azurewebsites.net/',
+  authority: environment.identityServer.IDENTITY_SERVER_BASE_URL,
```

Comment:
I think this should be just fine, do you have the environment config replacement thing in workspace.json?
Also not sure if it matters but you had originally a trailing forward slash and now it is missing.

Reply:
@pshomov 
Yes we have the file replacement line in our workspace.json.

Ill double check the trailing slash. 

Trying to load this env variable locally from the environment file by pointing towards process.env.IDENTITY_SERVER_BASE_URL in environments.ts doesnt work. But maybe its just not working locally for once :P

Reply:
if you want to try it locally then you should rename your .prod file to be your default one and then try it. Alternatively you can `yarn build` your project with `--prod` flag to make it build a prod distro and run it directly.